app/controllers/api/v0/manifests_controller.rb

```diff
@@ -12,30 +8,31 @@ def search
   end
 
   def validate
-    begin
-      manifest_content = JSON.parse(request.body.read)
+    manifest_content = read_body_as_json
+    unless performed?
```

Comment:
oh interesting - where might we have already returned an error msg in this case? Perhaps writing a test to cover that case would clarify the need to call `performed?`